istioctl/pkg/auth/testdata/rbac-policies-services-only-v2.golden.yaml

```diff
@@ -29,3 +17,37 @@ spec:
     labels:
       app: productpage
 ---
+apiVersion: rbac.istio.io/v1alpha1
+kind: AuthorizationPolicy
+metadata:
+  name: bind-all-services
+  namespace: default
+spec:
+  allow:
+  - role: all-services
+    subjects:
+    - names:
+      - foo
+---
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRole
+metadata:
+  name: service-viewer
+  namespace: default
+spec:
+  rules:
+  - constraints:
+    - key: destination.labels[version]
```

Comment:
Just a reminder for the some future change in v2: I think we will not only `destination.labels` in constraints, and only allow it in the `workloadSelector`

Reply:
Thanks for the update. Does that mean we will move stuff from `destination.labels` to the `selector` as well?

Reply:
Let's do that in the next PR. This PR is long enough.